Dockerfile

```diff
@@ -56,6 +56,5 @@ RUN GOARCH=$TARGETARCH go install -tags=xversion -ldflags="\
       -X tailscale.com/version.GitCommit=$VERSION_GIT_HASH" \
       -v ./cmd/tailscale ./cmd/tailscaled
 
-FROM alpine:3.14
-RUN apk add --no-cache ca-certificates iptables iproute2 ip6tables
```

Comment:
Why remove these deps? Tailscale still needs them?

Reply:
technically we don't need ca-certificates or iproute2 anymore. (but doesn't hurt to keep 'em)

but definitely need iptables and ip6tables still.